packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -321,6 +335,59 @@ export class GrpcPlugin extends BasePlugin<grpc> {
     return (original as any).call(self, call);
   }
 
+  private _patchLoadPackageDefinition() {
+    const plugin = this;
+    return (original: typeof grpcTypes.loadPackageDefinition) => {
+      plugin._logger.debug('patching loadPackageDefinition');
+      return function loadPackageDefinition(
+        this: grpc,
+        packageDef: grpcTypes.PackageDefinition
+      ) {
+        const result = original.apply(this, arguments as never);
+        // Copied from exports.loadPackageDefintion(...)
+        for (const serviceFqn in packageDef) {
```

Comment:
Would it make sense to create a private function to do the code inside this `for` loop? That could help simplify the overall `_patchLoadPackageDefinition` function